spec/Section 5 -- Validation.md

```diff
@@ -1524,6 +1524,31 @@ query ($foo: Boolean = true, $bar: Boolean = false) {
 }
 ```
 
+### Stream Directives Are Used On List Fields
+
+**Formal Specification**
+
+* For every {directive} in a document.
+* Let {directiveName} be the name of {directive}.
+* Let {adjacent} be the AST node the directive affects.
+* If {directiveName} is "stream"
```

Comment:
Minor, but we might as well guide towards the more optimal code path. (Also I believe we use `:` after `If` statements?)

```suggestion
* If {directiveName} is "stream":
  * Let {adjacent} be the AST node the directive affects.
```